bedrock/settings/base.py

```diff
@@ -1739,6 +1742,9 @@ def JINJA_CONFIG():
         'source_filenames': (
             'js/libs/modernizr.custom.shiv-load.js',
             'js/base/mozilla-input-placeholder.js',
+            'js/libs/jquery.waypoints.min.js',
```

Comment:
There are multiple pages already using this lib since before the static changes. If there we're problems, we would have seen broken pages already. Plus this has all been tested on demo servers, so we should have picked up any problems.